Remove usage of callouts in configuration code blocks (#403)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/proc_creating-openstack-environment-file-for-multiple-clouds.adoc

ifdef::include_when_13[]
. In the `stf-connectors.yaml` file, configure the `MetricsQdrConnectors` address to connect to the {MessageBus} on the overcloud deployment. Configure the `CeilometerQdrEventsConfig`, `CeilometerQdrMetricsConfig`, and `CollectdAmqpInstances` topic values to match the AMQP address that you want for this cloud deployment.
endif::include_when_13[]
ifndef::include_when_13[]
. In the `stf-connectors.yaml` file, configure the `MetricsQdrConnectors` address to connect to the {MessageBus} on the overcloud deployment. Configure the `CeilometerQdrEventsConfig`, `CeilometerQdrMetricsConfig`, `CollectdAmqpInstances`, and `CollectdSensubilityResultsChannel` topic values to match the AMQP address that you want for this cloud deployment.
endif::include_when_13[]
+

> * The `resource_registry` configuration directly loads the collectd service because you are not including the `collectd-write-qdr.yaml` environment file for multiple cloud deployments.

Reviewer: ```suggestion
* The `resource_registry` configuration directly loads the collectd service because you do not include the `collectd-write-qdr.yaml` environment file for multiple cloud deployments.
```

ifdef::include_when_13,include_when_17[]
* Replace the `MetricsQdrConnectors.host` parameter with the value of `HOST/PORT` that you retrieved in xref:retrieving-the-qdr-route-address_assembly-completing-the-stf-configuration[].
* Replace the `caCertFileContent` parameter with the contents retrieved in xref:getting-ca-certificate-from-stf-for-overcloud-configuration_assembly-completing-the-stf-configuration[].
endif::include_when_13,include_when_17[]
* Set the `CeilometerQdrEventsConfig.topic` to define the topic for Ceilometer events. This value is the address format of `anycast/ceilometer/cloud1-event.sample`.
* Set the `CeilometerQdrMetricsConfig.topic` to define the topic for Ceilometer metrics. This value is the address format of `anycast/ceilometer/cloud1-metering.sample`.
* Set the `CollectdAmqpInstances` to define the topic for collectd events. This value is the format of `collectd/cloud1-notify`.
* Set the `CollectdAmqpInstances` to define the topic for collectd metrics. This value is the format of `collectd/cloud1-telemetry`.
ifndef::include_when_13[]

> * Set the `CollectdSensubilityResultsChannel` to define the topic for collectd-sensubility events. Ensure that this value is the exact string format `sensubility/cloud1-telemetry`

Reviewer: ```suggestion
```
Same as previous comment, we can delete

Reviewer 2: Same as previous comment I don't think it should be removed, it should just be like the others?

Reviewer: ```suggestion
* Set the `CollectdSensubilityResultsChannel` to define the topic for collectd-sensubility events. This value is the format of `sensubility/cloud1-telemetry`
```
